Add SUSE Liberty Linux 9 support

Bischoff opened the pull request:
Add SUSE Liberty Linux 9 support to 4.3 Build Validation

See https://github.com/uyuni-project/sumaform/pull/1259

srbarrios reviewed (approved):
I did not checked MACs I hope they are fine.
Also, I did not do matchs to see if we can really allocate 8GB more in the same hypervisor.

Bischoff commented:
> I did not checked MACs I hope they are fine.

They should be:
```
srv/salt/dhcpd-server/mgr_prv_suse_net/dhcpd.conf:host suma-bv-43-min-liberty9       { hardware ethernet aa:b2:92:42:00:c5; fixed-address 10.84.221.197; option host-name "suma-bv-43-min-liberty9";      }
srv/salt/dhcpd-server/mgr_prv_suse_net/dhcpd.conf:host suma-bv-43-minssh-liberty9    { hardware ethernet aa:b2:92:42:00:e5; fixed-address 10.84.221.229; option host-name "suma-bv-43-minssh-liberty9";   }
```
>  Also, I did not do maths to see if we can really allocate 8GB more in the same hypervisor.
